### ISSUE TITLE
CAT broke 'knock to unlock' app

### ISSUE THREAD

**Issue by MrZar0** [dokterdok/Continuity-Activation-Tool#129]
CAT is working just fine on my MBP 15" 2011 with an IOGEAR 4.0 usb dongle. 
handoff, instant hotspot, and airdrop are working just fine. 
my problem is with knock to unlock app, that after the installation of CAT is no longer working. 
the iphone won't connect with the mac (with the usb bluetooth dongle). what's the problem? maybe the kext patch? how can i solve it?
i've already reinstalled the iOS and OS X companion app, unplugged and plugged in again the dongle but nothing changes. 
please help, knock was very useful and time saving.
sorry for the bad english.

**Comment by dokterdok** [comment]
Hi, could you please verify if the steps described in #122 solve your issue and then  report back?

**Comment by MrZar0** [comment]
unfortunately it doesn't work. the wifi stop working so i've reinstalled CAT to solve the problem, now i'm back with the same issue, handoff and the other continuity features works, knock to unlock doesn't.

**Comment by samsawyer** [comment]
If it's helpful, I just encountered some version of this problem while using a early 2011 Macbook Pro 15" with the IOGear Bluetooth 4.0 USB adapter.

I was able to resolve it by uninstalling CAT, rebooting, running and pairing Knock, rebooting again, then reinstalling CAT and rebooting yet again. (I'm not sure the middle reboot was necessary.) 

Anyway, after doing that both Knock and Continuity/Handoff seem to be working. If there's anything I can supply to help in troubleshooting, I'd be happy to help.
